schemas/storage.json

```diff
@@ -63,7 +63,6 @@
           },
           {
             "name": "getBytesInUse",
-            "unsupported": true,
```

Comment:
As mentioned that the top of the readme, the source of truth for this information is https://searchfox.org/mozilla-central/rev/73adec8ae1089db082e350da8951b8dde7aa3bcb/toolkit/components/extensions/schemas/storage.json#66 so this change will get overwritten on the next update from Firefox sources. This project only exposes APIs once they are available in Firefox so since [Firefox doesn't support it for `local` and `managed` storage areas yet](https://bugzilla.mozilla.org/show_bug.cgi?id=1385832) then it won't be added here. Please reopen once it's implemented in Firefox.